Cargo.toml

```diff
@@ -115,8 +115,8 @@ goauth = { version = "0.8.1", optional = true }
 smpl_jwt = { version = "0.5.0", optional = true }
 
 # API
-async-graphql = { version = "2.4.3", optional = true }
-async-graphql-warp = { version = "2.4.3", optional = true }
+async-graphql = { version = "2.4.10", optional = true }
```

Comment:
This bump was required for https://github.com/async-graphql/async-graphql/issues/387